src/sync/models/centralizedOracle.js

```diff
@@ -35,7 +35,7 @@ class CentralizedOracle {
 
   translate() {
     return {
-      address: this.oracle,
+      address: this.contractAddress,
```

Comment:
@dwalintukan can u check whether this is correct?

Reply:
This is correct. I checked the mapping.